src/js/store/index.js

```diff
@@ -0,0 +1,36 @@
+import { makeLotto } from '../utils/lotto.js';
+import reactive from '../utils/reactive.js';
+
+const { state, proxyState } = reactive({
+  amount: null,
+  lottoList: [],
+  isVisibleLottos: false,
+});
+
+const getter = {
+  getLottoAmount() {
+    return Math.floor(state.amount / 1000);
+  },
+};
+
+const mutation = {
+  setAmount(payload) {
+    state.amount = payload;
+  },
+  setLottoList(payload) {
+    state.lottoList = payload;
+  },
+  setIsVisibleLottos(payload) {
+    state.isVisibleLottos = payload;
+  },
+};
+
+const action = {
+  makeLottoList() {
+    const lottoAmount = getter.getLottoAmount();
+    const lottoList = new Array(lottoAmount).fill(null).map(() => makeLotto());
+    mutation.setLottoList(lottoList);
+  },
+};
+
+export { mutation, proxyState, action, getter };
```

Comment:
```suggestion
export { mutation, state: Object.freeze(state), action, getter };
```
Object.freeze를 이용해서 setter를 실행하지 못하도록 하는 방법도 있답니다!

Reply:
감사합니다!! 적용하였습니다!!